src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -152,14 +182,21 @@ public void systemLoadsManageConditionsPage() {
 		assertTrue(conditionsPage.containsText("Conditions"));
 	}
 
-	@When("a user clicks on Add new condition")
+	@And("a user clicks on Add new condition")
 	public void userClicksAddNewCondition() {
 		conditionPage = (ConditionPage) conditionsPage.clickOnAddNewCondition().waitForPage();
 	}
 
 	@And("a user enters patient condition")
 	public void enterExistingCondition() {
 		conditionPage.typeInCondition(CONDITION);
+		conditionPage.clickSave();
```

Comment:
how does it sound if the method is renamed enterFirstCondition()?

Reply:
> enterFirstCondition()

@kdaud this method has to first save for it to add another one.... try it on the ui to see whether it makes sense